packages/wonder-blocks-dropdown/components/multi-select-menu.js

```diff
@@ -69,14 +69,14 @@ type Props = {
      * Optional styling to add.
      */
     style?: any,
-};
+|};
 
-type State = {
+type State = {|
```

Comment:
Originally I was thinking of only making `Props` exact objects, but the eslint rule doesn't have a way to do that and there doesn't seem to be a downside from making `State` exact as well.

Reply:
I like making state exact. It really helps remind me to remove things that are no longer used.